misc-tools/dj_make_chroot.in

```diff
@@ -262,19 +265,16 @@ cp -r /usr/share/ca-certificates/* "$CHROOTDIR/usr/share/ca-certificates" || tru
 cp -r /usr/local/share/ca-certificates/* "$CHROOTDIR/usr/local/share/ca-certificates" || true
 
 if [ "$DISTRO" = 'Debian' ]; then
+	if [ "x$RELEASE" != 'xtesting' ]; then
```

Comment:
Should it also not be disable for sid/unstable?

Reply:
`sid` was already excluded on line 41. I'm not sure whether we should change that.